swedish-oidc-profile.md

```diff
@@ -374,31 +374,26 @@ The OpenID Provider MUST NOT release any user identity claims other than the man
 
 OpenID Providers MUST return claims on a best effort basis. However, an OpenID Provider asserting it can provide a user claim does not imply that this data is available for all its users. Relying Parties MUST be prepared to receive partial data. 
 
-An OpenID Provider compliant with this profile MUST NOT release any identity claims<sup>1</sup> in the ID token, or via the UserInfo endpoint, if they have not been explicitly requested via the `scope` and/or `claims` request parameters, or by a policy<sup>2</sup> known, and accepted, by the involved parties. 
+An OpenID Provider compliant with this profile MUST NOT release any identity claims<sup>1</sup> in the ID token, or via the UserInfo endpoint, if they have not been explicitly requested via the `scope` and/or `claims` request parameters, or by a policy known, and accepted, by the involved parties. Furthermore, an OpenID Provider MUST NOT release any claims to the Relying Party which it has not been
+authorized to receive. 
```

Comment:
Should be more like this:
"... an OpenID Provider MUST NOT release any claims to a Relying Party that has not been authorised to receive them."

Reply:
Yes, but I also wanted to point out that an OP must not deliver claims that has not been explicitly requested. So you think that this requirement is not something we should state?

Reply:
Or is your comment just a comment on the last sentence?